venafi/provider.go

```diff
@@ -143,6 +146,12 @@ Example:
 				DefaultFunc: schema.EnvDefaultFunc(envVenafiClientID, defaultClientID),
 				Description: "application that will be using the token",
 			},
+			providerSkipRetirement: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc(envVenafiSkipRetirement, defaultSkipRetirement),
+				Description: `Skips the retirement certificate on the configured Venafi Platform. By default this is set to false enabling the retirement certificate functionality`,
```

Comment:
Description: Skips the retirement of certificates on terraform destroy on Venafi Platform's side. Default is false.

Reply:
Skips the retirement of current certificate resource in its Venafi Platform during Terraform destroy operation. Set `false` by default/Default value is `false`